### PR TITLE
[15.0] [MIG] sale_line_returned_qty_mrp

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

Depends on:

- https://github.com/OCA/stock-logistics-workflow/pull/1080

### CODE REVIEW

**Review by JuanyDForgeflow — APPROVED** [review]
Code review LGTM

**Review by MiquelRForgeFlow — APPROVED** [review]

File: sale_line_returned_qty_mrp/models/sale_order_line.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ class SaleOrderLine(models.Model):
     )
     def _compute_qty_returned(self):
         # Based on the standard method in 'sale_mrp' to compute qty_delivered
-        super(SaleOrderLine, self)._compute_qty_returned()
+        res = super(SaleOrderLine, self)._compute_qty_returned()
```
Comment (rousseldenis) [inline sale_line_returned_qty_mrp/models/sale_order_line.py]: You can use short notation

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Minor remark.

Moreover, should be great to add tests for uncovered cases.

**Comment by MiquelRForgeFlow** [comment]
Should we wait https://github.com/OCA/stock-logistics-workflow/pull/1081 to be finished before merging this PR? 🤔

**Comment by rousseldenis** [comment]
> Should we wait #1081 to be finished before merging this PR? thinking

Better

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**